playwright.config.ts

```diff
@@ -17,9 +17,7 @@ const config: PlaywrightTestConfig = {
   // Run your local dev server before starting the tests:
   // https://playwright.dev/docs/test-advanced#launching-a-development-web-server-during-the-tests
   webServer: {
-    command: process.env.CI
-      ? 'NODE_OPTIONS="--dns-result-order=ipv4first" npm run start'
-      : 'npm run build && NODE_OPTIONS="--dns-result-order=ipv4first" npm run start',
+    command: 'npm run start',
```

Comment:
Still need to call build right?

Reply:
For CI, we have the build step before running e2e so it's taken care of. For local runs, maybe we can add a `prestart` script similar to `preinstall` that will display a message indicating that the build needs to happen first...